README.md

```diff
@@ -142,42 +142,60 @@ Once the libraries are added to a project, the basics of adding and reading
 a document are:
 
 ```java
-import com.cloudant.sync.datastore.DatastoreManager;
-import com.cloudant.sync.datastore.Datastore;
-import com.cloudant.sync.datastore.DatastoreNotCreatedException;
-import com.cloudant.sync.datastore.DocumentBodyFactory;
-import com.cloudant.sync.datastore.DocumentException;
-import com.cloudant.sync.datastore.DocumentRevision;
-import com.cloudant.sync.datastore.UnsavedFileAttachment;
```

Comment:
I don't think we should remove the import statements, it makes it easier to for users to see where the classes (outside of the JDK) are coming from.